schedule/yast/maintenance/lvm_thin_provisioning.yaml

```diff
@@ -1,17 +1,18 @@
 ---
-name: lvm_thin_provisioning_dev
+name: lvm_thin_provisioning
 vars:
   DUD_ADDONS: sdk
   YUI_REST_API: 1
 schedule:
   suggested_partitioning:
     - installation/partitioning/new_partitioning_gpt
-  default_systemd_target:
-    - installation/installation_settings/validate_default_target
+  grub:
```

Comment:
grub handling will be always part of the default not the test case (this yaml I mean).